python/paddle/fluid/framework.py

```diff
@@ -3503,7 +3503,12 @@ def build_adjacency_list(self):
             wrapped_adj_list[IrNode(k)] = {IrNode(n) for n in v}
         return wrapped_adj_list
 
-    def draw(self, save_path, name, marked_nodes=None, remove_ctr_var=True):
+    def draw(self,
```

Comment:
IMO the previous approach could be preserved here. The .pdf files could be generated automatically, without the `--pdf` option.